packages/mobile/src/paymentRequest/PaymentRequestReviewCard.tsx

```diff
@@ -36,42 +38,64 @@ const mapStateToProps = (state: RootState): StateProps => {
 }
 
 // Bordered content placed in a ReviewFrame for summarizing a payment request
-class PaymentRequestReviewCard extends React.Component<OwnProps & StateProps & WithNamespaces> {
-  render() {
-    const {
-      recipient,
-      address,
-      e164PhoneNumber,
-      t,
-      dollarBalance,
-      currency,
-      value,
-      comment,
-    } = this.props
+function PaymentRequestReviewCard({
```

Comment:
This is a non-trivial component. Should we memoize? Does redux's checks for if a component should re-render still work for FCs?

Reply:
I think we're fine.
Changing it to a FC didn't affect the re-render. This component is still wrapped by `connect` and only re-renders when the mapped props actually change ([based on shallow equality comparisons](https://react-redux.js.org/api/connect#mapstatetoprops-state-ownprops-object)).
Internally it does contain a new hook which also listens to redux, but again it will only re-render if the value changes ([based on `===` reference comparison](https://react-redux.js.org/api/hooks#equality-comparisons-and-updates))